### ISSUE TITLE
TikTok playback broken: Failed to find data for tiktok video

### ISSUE THREAD

**Issue by vSilva-gb** [DuncteBot/skybot-lavalink-plugin#7]
## Description
When I try to play any tiktok video it returns "Failed to find data for tiktok video".
". Full logs here

## Version info
**Client used:** erela.js

Output of `java -version`:
```
2022-06-21 03:28:02.631 ERROR 27 --- [ader-2-thread-1] l.server.player.AudioLoaderRestHandler : Track loading failed
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Failed to find data for tiktok video
at com.dunctebot.sourcemanagers.tiktok.TikTokAudioSourceManager.extractData(TikTokAudioSourceManager.java:139) ~[skybot-lavalink-plugin-1.3.3.jar!/:na]
at com.dunctebot.sourcemanagers.tiktok.TikTokAudioSourceManager.extractData(TikTokAudioSourceManager.java:98) ~[skybot-lavalink-plugin-1.3.3.jar!/:na]
at com.dunctebot.sourcemanagers.tiktok.TikTokAudioSourceManager.loadItem(TikTokAudioSourceManager.java:74) ~[skybot-lavalink-plugin-1.3.3.jar!/:na]
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:436) ~[walkyst-lavaplayer-bded96a.jar!/:na]
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:418) ~[walkyst-lavaplayer-bded96a.jar!/:na]
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:217) ~[walkyst-lavaplayer-bded96a.jar!/:na]
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[na:na]
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[na:na]
at java.base/java.lang.Thread.run(Thread.java:831) ~[na:na]
Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation:
lavaplayer.version: 1.3.97-original
os.arch: amd64
os.name: Linux
os.version: 5.4.0-117-generic
java.vendor: Oracle Corporation
java.version: 16
java.runtime.version: 16+36-2231
java.vm.version: 16+36-2231
```

**Comment by duncte123** [comment]
is it pornhub or tiktok that is broken?

**Comment by vSilva-gb** [comment]
Sorry, TikTok is broken

**Comment by duncte123** [comment]
Thank you for the clarification, I'll have a look

**Comment by duncte123** [comment]
@vSilva-gb There should be some HTML dumped to the logs before that exception hits, do you have that for me?

**Comment by duncte123** [comment]
Fixed in version 1.4.0
